tslib/src/functions/check-receiver/check-receiver.ts

```diff
@@ -18,12 +28,20 @@ export class CheckReceiver {
 
     await tableService.insertEntityAsync('receivedCheck', receivedCheckEntity)
 
-    const receivedMessage: ICheckNotificationMessage = {
-      version: 1,
-      checkCode: receivedCheck.checkCode,
-      notificationType: CheckNotificationType.checkReceived
+    // as per #48506 - check-receiver will now handle this event instead of check-notifier-batch
+    const request = this.checkNotifierDataService.createCheckReceivedRequest(receivedCheck.checkCode.toLowerCase())
+
+    try {
+      await this.checkNotifierDataService.executeRequestsInTransaction([request])
+    } catch (error) {
+      context.log.error(`failed to write check received notification to database for check ${receivedCheck.checkCode}\n Falling back to check notification queue message.`)
```

Comment:
For consistency with other Functions, and to make searching for logs in AI easier, please consider prefix this log message with the function name.